core/src/main/scala/org/ensime/util/FileUtil.scala

```diff
@@ -84,14 +84,14 @@ object FileUtils {
     }
   }
 
-  def writeDiffChanges(changes: List[FileEdit], cs: Charset): Either[Exception, File] = {
+  def writeDiffChanges(changes: List[FileEdit])(implicit cs: Charset): Either[Exception, File] = {
     //TODO: add support for NewFile and DeleteFile
```

Comment:
should we create separate issue for that? I haven't yet looked which of our refactorings can create/delete files and in how many cases.